test/pseudo-tty/test-handle-wrap-isrefed-tty.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+// see also test/parallel/test-handle-wrap-isrefed.js
+
+const common = require('../common');
+const strictEqual = require('assert').strictEqual;
+
+function makeAssert(message) {
+  return function(actual, expected) {
+    strictEqual(actual, expected, message);
+  };
+}
+
+const assert = makeAssert('hasRef() not working on tty_wrap');
+
+const ReadStream = require('tty').ReadStream;
+const tty = new ReadStream(0);
+const isTTY = process.binding('tty_wrap').isTTY;
+assert(isTTY(0), true);
+assert(Object.getPrototypeOf(tty._handle).hasOwnProperty('hasRef'), true);
+assert(tty._handle.hasRef(), true);
+tty.unref();
+assert(tty._handle.hasRef(), false);
+tty.ref();
+assert(tty._handle.hasRef(), true);
```

Comment:
I agree with @addaleax ... we've been making an effort to convert as many tests as possible to using the strict assertion form where possible. As long as we're in this file making changes, we should go ahead and make the switch here. We can fix up the other tests for consistency separately